x-pack/packages/security/authorization_core/kibana.jsonc

```diff
@@ -1,5 +1,9 @@
 {
   "type": "shared-server",
   "id": "@kbn/security-authorization-core",
-  "owner": "@elastic/kibana-security"
-}
+  "owner": [
+    "@elastic/kibana-security"
+  ],
+  "group": "platform",
+  "visibility": "private"
```

Comment:
__question:__ I assume there won’t be any issue if in the future we decide to change visibility from `private` to `shared`, right?

Reply:
It shouldn't be a problem. See discussion on a [similar question](https://github.com/elastic/kibana/pull/199630#discussion_r1847748548).